tests/test_module_state.py

```diff
@@ -19,7 +19,7 @@
 def find_all_modules():
     for module in MODULES_PATH.listdir(sort=True):
         module = module.basename.replace('.py', '')
-        if module != '__init__':
```

Comment:
`if module.basename.endswith(".py"):` ?

Reply:
Oh, that does not work with `__init__.py`. Darn.

Reply:
that would also work, yes

Reply:
What about a regexp match:
`r"\([^-].*\)\.py"` and the first matching group is what you want anyway.